hr_holidays_validity_date/migrations/12.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    cr = env.cr
+    if openupgrade.column_exists(
+            cr,
+            'hr_leave_type',
+            'date_start'
+    ):
+        openupgrade.logged_query(
+            cr, """
+            UPDATE hr_leave_type
+            SET validity_start = date_start, validity_stop = date_end;
```

Comment:
but:
`validity_start` and `validity_stop` are of type **date**
and `date_start` and `date_end` were of type **datetime**

Reply:
Is this an issue?

Reply:
The time part will be lost but I dont think its very relevant and there is not much we can do with this anyways.
`date_start` and `date_end` columns are not dropped so if someone really needs it, the information is still there.

Reply:
it is ok to me.